package co.morillas.auth.core.exception

class UserNotFoundException(): RuntimeException("Username not found")